src/Microsoft.Framework.ConfigurationModel.CommandLine/CommandLineConfigurationExtension.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Framework.ConfigurationModel.CommandLine
+{
+    public static class CommandLineConfigurationExtension
```

Comment:
I think the extension class names should be plural, i.e. `CommandLineConfigurationExtension`. Please check the other extension method types in the PR for fixing.